voice-search/voice-search-impl/src/main/java/com/duckduckgo/voice/impl/listeningmode/VoiceSearchActivity.kt

```diff
@@ -68,10 +66,11 @@ class VoiceSearchActivity : DuckDuckGoActivity() {
     }
 
     private fun configureViews() {
-        window.setFlags(
-            WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS,
-            WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS,
-        )
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
```

Comment:
By convention we use `appBuildConfig` instead of `Build.VERSION.SDK_INT` and suppress annotation.

```
    @SuppressLint("NewApi")
    private fun configureViews() {
        if (appBuildConfig.sdkInt >= Build.VERSION_CODES.R) {
        ...
```